apps/frontend/src/pages/project/libs/components/contributors-list/contributors-list.tsx

```diff
@@ -13,12 +14,13 @@ const ContributorsList = ({
 	contributors,
 	isLoading,
 }: Properties): JSX.Element => {
+	const hasContributors = contributors.length > EMPTY_LENGTH;
+
 	return (
 		<div className={styles["container"]}>
 			<h2 className={styles["title"]}>Contributors</h2>
-			{isLoading ? (
-				<Loader />
-			) : (
+			{isLoading && <Loader />}
+			{!isLoading && hasContributors && (
```

Comment:
How about returning <Loader/> before, and don't use so many conditions for each block?

Reply:
Looks like a nice solution, let's see what will Liza say about it. The only thing is that we won't have this layout of a component at all (container and title). I just tried to do it as we've done before in table and projects page.

Reply:
I think in this case we will miss the title, so I would leave it